packages/ai/src/ai/account/store.py

```
@@ -761,7 +491,7 @@ def _get_default_storage_url() -> str:
         try:
             # Try to get user's home directory (works on Windows/Linux/Mac)
             home = Path.home()
-            storage_path = home / '.rocketlib' / 'dtc'
+            storage_path = home / '.rocketlib' / 'store'
         except Exception:
```

Ruff (0.15.7)

[warning] Do not catch blind exception: `Exception`

(BLE001)